builtin/logical/pki/crl_util.go

```diff
@@ -151,10 +155,16 @@ func buildCRL(b *backend, req *logical.Request) error {
 			return errutil.InternalError{Err: fmt.Sprintf("Unable to parse stored revoked certificate with serial %s: %s", serial, err)}
 		}
 
-		revokedCerts = append(revokedCerts, pkix.RevokedCertificate{
+		// NOTE: We have to change this to UTC time because the CRL standard
+		// mandates it but Go will happily encode the CRL without this.
+		newRevCert := pkix.RevokedCertificate{
 			SerialNumber:   revokedCert.SerialNumber,
-			RevocationTime: time.Unix(revInfo.RevocationTime, 0),
-		})
+			RevocationTime: time.Unix(revInfo.RevocationTime, 0).UTC(),
```

Comment:
The value you get from time.Time.Unix is always relative to UTC. The issue is that time.Unix parses from number of seconds since UTC into a local time, not UTC time. So this is just a conversion.

I didn't need to add the time.Time field but it feels better, with our further experience on the matter, to store a time. Time directly for our internal purposes.